pkg/metaserver/agent/pod/pod.go

```diff
@@ -216,6 +217,18 @@ func (w *podFetcherImpl) GetPod(ctx context.Context, podUID string) (*v1.Pod, er
 	return nil, fmt.Errorf("failed to find pod by uid %v", podUID)
 }
 
+func (w *podFetcherImpl) GetPodNoCache(ctx context.Context, podUID string) (*v1.Pod, error) {
```

Comment:
Why not just use GetPod with context bypass_cache=true?

Reply:
It's much better to use this function,  I haven't recognized it before.

Reply:
done